### ISSUE TITLE
plugin broken: file doesn't exist

### ISSUE THREAD

**Issue by sycured** [hCaptcha/hcaptcha-wordpress-plugin#73]
I did the update to 1.8.0, and I've this error:
```
FastCGI sent in stderr: "PHP message: PHP Warning:  require_once(/var/www/wordpress/wp-content/plugins/hcaptcha-for-forms-and-more/vendor/autoload.php): failed to open stream: No such file or directory in /var/www/wordpress/wp-content/plugins/hcaptcha-for-forms-and-more/hcaptcha.php on line 50
```

**Comment by csalzano** [comment]
1.8.0 also bricks my site with this error

```
Warning: require_once(/public_html/coreysalzano.com/wp-content/plugins/hcaptcha-for-forms-and-more/vendor/autoload.php): failed to open stream: No such file or directory in /public_html/coreysalzano.com/wp-content/plugins/hcaptcha-for-forms-and-more/hcaptcha.php on line 50

Fatal error: require_once(): Failed opening required '/public_html/coreysalzano.com/wp-content/plugins/hcaptcha-for-forms-and-more/vendor/autoload.php' (include_path='.:/usr/local/php73/pear') in /public_html/coreysalzano.com/wp-content/plugins/hcaptcha-for-forms-and-more/hcaptcha.php on line 50
```

**Comment by e271828-** [comment]
@csalzano @sycured sorry about that! We just rolled back: update to 1.6.5 and you should be set.

**Comment by e271828-** [comment]
Closed in db9c04ed2efaf1c208ab6717e5db5110c631df4f
